npm/next/5.0.0/config.n4jsd

```diff
@@ -0,0 +1,6 @@
+export default external public function getConfig(): ~Object with {
+    serverRuntimeConfig: Object;
+    publicRuntimeConfig: Object;
+};
+
+/* -*- mode: typescript;-*- */
```

Comment:
repo has colouring configured, what is the point of copy&pasting this comment?
https://github.com/NumberFour/n4jsd/blob/master/.gitattributes#L1

Reply:
there is no point, was just taking an existing n4jsd file as a blueprint...